### DISCUSSION TITLE
EA App Install Unsuccessful

### DISCUSSION THREAD

**Discussion by MagnusAustralia** [moraroy/NonSteamLaunchers-On-Steam-Deck#114] [Q&A]
Hello today when I went to open the EA app I noticed tat it got stuck as a gray screen in the login screen dimensions, 

As this didn't work I reinstalled EA through this program but that unfortunately didn't fix it. I then did a "Start Fresh" with that being unsuccessful and I just did a factory reboot with the EA App still getting stuck on the login page with a pure gray coloration. I know that the EA system still worked as I was able to play EA games connected to my Steam account. 

I was wondering if anyone has any recommendations. 
Thanks in advance :smile: 

P. S. There is a good chance that this could be in some way a problem with EA as I also tried installing the EA App without the use of  this program and got similar results.

**Comment by moraroy** [comment]
@williamcool7890 it may have to do with the current proton.. it wasn't like this a couple versions back. It's installing just not opening all the way for some reason.

**Comment by moraroy** [comment]
@williamcool7890 this has since been fixed with the current .sh file. Geproton 8-14 has fixed this issue and the script now runs that. I'll be closing this.
